src/navigations/HomeBottomTabs.js

```diff
@@ -65,21 +67,21 @@ function HomeBottomTabs() {
     }
     if (route.name === 'Feed') {
       return (
-        <View style={styles.center}>
+        <View style={styles.iconContainer}>
           <MemoFeed fill={color} />
         </View>
       );
     }
     if (route.name === 'News') {
       return (
-        <View>
+        <View style={styles.iconContainer}>
           <MemoNews fill={color} />
         </View>
       );
     }
 
     return (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the style of the icon container has been changed from `styles.center` to `styles.iconContainer` for the 'Feed' and 'News' routes as well. This maintains consistency in the styling of the icon containers across different routes.
```diff
- <View style={styles.center}>
+ <View style={styles.iconContainer}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->